src/components/BioButton/scaffolding/BioButton.hbs

```diff
@@ -1,5 +1,9 @@
-<bio-button title="Watch" data-resources="[{paths: ['components/BioButton/index.js']}]" url="https://www.biotope.sh"></bio-button>
-<bio-button title="Watch" modifier="whiteText" data-resources="[{paths: ['components/BioButton/index.js']}]" url="https://www.biotope.sh"></bio-button>
-<bio-button title="Who's behind Biotope" modifier="long" data-resources="[{paths: ['components/BioButton/index.js']}]" url="https://www.biotope.sh"></bio-button>
-<bio-button title="Github" modifier="blue" data-resources="[{paths: ['components/BioButton/index.js']}]" url="https://www.biotope.sh"></bio-button>
-<bio-button title="Contact" modifier="full" data-resources="[{paths: ['components/BioButton/index.js']}]" url="https://www.biotope.sh"></bio-button>
+<bio-button title="Watch" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
+<bio-button title="Watch" modifier="whiteText" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
+<bio-button title="Who's behind Biotope" modifier="long" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
+<bio-button title="Github" modifier="blue" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
```

Comment:
is there any use for the modifier='blue'?, because there is no style 'blue' defined in the bio.buttons scss

Reply:
Correct. I removed it